charts/celeborn/values.yaml

```diff
@@ -53,28 +53,28 @@ volumes:
     - mountPath: /mnt/celeborn_ratis
       hostPath: /mnt/celeborn_ratis
       type: hostPath
-      size: 1Gi
+      capacity: 1Gi
   worker:
     - mountPath: /mnt/disk1
       hostPath: /mnt/disk1
       type: hostPath
       diskType: HDD
```

Comment:
Shall we set default diskType to SSD? As we already set default value of `celeborn.worker.flusher.threads` to 16, the same with ssd's flusher threads

Reply:
SGTM